.github/workflows/incident-classification.yml

```diff
@@ -0,0 +1,107 @@
+            Which third-party services were involved. Choose from:
+            - **cloudflare** - Including R2, Workers
+            - **aws** - Including Lambda, CloudWatch, etc.
+            - **fastly** - Including Image Optimizer
+            - **github** - GitHub API or services
+            - **microsoft** - Azure, SharePoint, OneDrive
+            - **unpkg** - unpkg.com CDN
+            - **zscaler** - zScaler proxy/security
+            - **null** - If no external vendor was involved
+
+            ### 3. rootCause (string)
+            Primary cause category. Choose ONE from:
+            - **third-party-outage** - External service disruption
+            - **configuration-change** - Configuration error or misconfiguration
+            - **deployment-issue** - Code deployment or release problem
+            - **resource-limits** - Capacity, concurrency, or resource exhaustion
+            - **credential-issue** - Authentication, authorization, or credential problems
+            - **dns-issue** - DNS resolution or configuration problems
+            - **network-issue** - Network connectivity or routing issues
+            - **unknown** - Cause not identified or maintenance
+
+            ## Review Instructions:
+
+            1. **Check for missing classifications**: Identify any incidents that are missing one or more
+               of the required fields (affectedComponents, externalVendors, rootCause).
+
+            2. **Validate classifications**: For each classified incident, verify that:
+               - The affectedComponents are accurate based on the incident name and message
+               - The externalVendors are correctly identified from the incident description
+               - The rootCause is the most appropriate category
+
+            3. **Suggest corrections**: If any classifications appear incorrect or incomplete, provide
+               specific suggestions using the ```suggestion markdown syntax for inline comments.
+
+            4. **Auto-classify if needed**: If there are new incidents without classifications, analyze
+               the incident name and message to determine the appropriate values for all three fields,
+               and provide the suggested classifications.
+
+            5. **Look for patterns**: Comment on any trends you notice (e.g., multiple incidents from
+               the same vendor, recurring root causes).
+
+            Please provide thorough feedback with inline comments on:
+            - Missing or incomplete classifications
+            - Potentially incorrect classifications
+            - Suggested improvements to the taxonomy values
+
+            Use the ```suggestion markdown syntax to provide actionable recommendations that can be
+            merged directly into the JSON file.
+
+          claude_args: |
+            --allowedTools "mcp__github_inline_comment__create_inline_comment,Bash(gh pr comment:*), Bash(gh pr diff:*), Bash(gh pr view:*)"
```

Comment:
This is also not consistent, we must allow the agent to make git adds and git commits